app/components/link_component.html.erb

```diff
@@ -1,13 +1,13 @@
 <%= link_to href, **merged_opts do %>
   <% if icon && (icon_position != "right") %>
-    <%= lucide_icon(icon, class: icon_classes) %>
+    <%= helpers.icon(icon, class: icon_classes) %>
```

Comment:
Is this `helpers.icon()` the right way of doing this? It seems like anything outside of the view doesn't have access to `icon` directly, but not super familiar with the ruby scoping here.

Reply:
Yep, `helpers.icon` is the correct way!